jest.config.typescript.js

```diff
@@ -24,7 +24,7 @@ module.exports = {
     // collectCoverageFrom: null,
 
     // The directory where Jest should output its coverage files
-    coverageDirectory: 'reports/coverage',
+    coverageDirectory: 'reports/ts-coverage',
```

Comment:
You'd have to add this coverage directory to the sonar properties file. Also, we need to update the GitHub workflow (report-code-coverage.yml), where we update the filesystem paths.

Additionally, we need to upload (report-code-coverage.yml) the new coverage reports to Codecov.

Reply:
updated